pkg/tool/git.go

```diff
@@ -67,8 +67,8 @@ func (g Git) ListChangedFilesInDirs(commit string, dirs ...string) ([]string, er
 	return strings.Split(changedChartFilesString, "\n"), nil
 }
 
-func (g Git) GetUrlForRemote(remote string) (string, error) {
-	return g.exec.RunProcessAndCaptureOutput("git", "ls-remote", "--get-url", remote)
+func (g Git) GetUrlForRemote(ref string) (string, error) {
+	return g.exec.RunProcessAndCaptureOutput("git", "ls-remote", "--get-url", ref)
```

Comment:
I don't think this will work... we may need to keep the `--remote` flag after all since it's possible that multiple remotes will have the same ref.

Reply:
Indeed. Will restore `--remote`.

Reply:
Let's get @unguiculus's input before doing too much more work.

Reply:
I will have to think this through. A `--since` flag probably makes sense, but we need to keep `--remote` for now.